tests/test_cluster.py

```diff
@@ -1478,5 +1478,27 @@ def test_minimal(self):
                                                     }
                                                 })
 
+class TestClusterManageCommands(QdsCliTestCase):
```

Comment:
Can you add some more negative tests:
- fail if both id and label are provided.
- fail if private_dns is provided with add_node.
- fail if private_dns is not provided with remove_node/update_node